src/lib/shell/BUILD.gn

```
@@ -58,6 +58,8 @@ static_library("shell") {
       "MainLoopDefault.cpp",
       "streamer_k32w.cpp",
     ]
+  } else if (chip_device_platform == "nrfconnect") {
```

```suggestion
  } else if (current_os == "zephyr") {
```? Now that we have the Telink platform, too.